x/staking/types/authz_authorization.go

```diff
@@ -0,0 +1,135 @@
+package types
```

Comment:
isn't this filename a bit redundant? what about just `authorization.go` or `authz.go`?